python/pyspark/sql/tests/connect/test_parity_dataframe.py

```diff
@@ -41,6 +41,11 @@ def test_observe(self):
     def test_observe_str(self):
         super().test_observe_str()
 
+    # TODO(SPARK-XXXXX): Support Structured Streaming
```

Comment:
This test does not use streaming.

Reply:
Yeah, I deferred implementing API in Spark connect because we don't support withWatermark now, but we wouldn't need to mention about streaming. We could just say it's not implemented yet.

Reply:
I just updated the TODO comment with actual JIRA ticket number.